src/gen/conditions.rs

```diff
@@ -718,6 +737,13 @@ pub fn parse_conditions(
                 }
                 spend.birth_height = Some(h);
             }
+            Condition::AssertEphemeral => {
```

Comment:
Hey, when we parse conditions and put them into the `state` set, do we store ancillary data like hints in another table? Does it depend on the `Condition`?

What I'm getting at is, is each of the conditions passed up (IF it is passed up), identical to the `Condition` that came out of the `Spend` ?

Reply:
there is no other table. We store just what the full node needs. So the create coin hint is one of those things, but not memos. The conditions parser is quite full-node centric right now, it deliberately drops any information that's not strictly required for consensus (with the exception of the coin hint, which is required for a full-node service).

I don't think I understand what you mean. The `Condition` enum (aka variant) are the parsed-out conditions and their arguments. Consensus rules allows any number of extra arguments to the conditions, but we only keep the ones whose meaning is defined.

There's a next step where these conditions are interpreted and collated/deduplicated. Since many coins may have (different) `ASSERT_HEIGHT_ABSOLUTE` conditions, we really only care about the most restrictive one. So we gather up all the most restrictive conditions and return that back up (in `SpendBundleConditions`, which contains a list of `Spend` object with conditions for each coin we're spending).